### PR TITLE
Standard logger with explicit format for parsing

### PR DESCRIPTION
**Pull request by juanignaciosl**
Fixes CartoDB/cartodb-platform/issues/4780 .

### CODE REVIEW

**Comment by juanignaciosl** [comment]
As this is pretty low-level, I'd like to ask for both CR and acceptance to backend.

**Comment by juanignaciosl** [comment]
retest this please

File: config/initializers/carto_db.rb [hunk 1]
```diff
@@ -288,4 +288,25 @@ def self.get_absolute_url(url)
   rescue
     nil
   end
+
+  def self.standard_logger(log_file_path)
```
Comment (javitonino) [inline config/initializers/carto_db.rb]: I don't like the name `standard` very much. Maybe something like `plain_logger` or `unformatted_logger` to make more clear it's purpose?

Just naming, so feel free to ignore.

**Comment by ethervoid** [comment]
@javitonino changed, second review?

**Comment by javitonino** [comment]
Found some more missing files that were not mentioned in the ticket:
- `synchronizations.log`
- `geocodings.log`

Could you use the new logger for those as well? :pray:

**Comment by ethervoid** [comment]
Sure! I added imports and I missed those

**Comment by ethervoid** [comment]
@javitonino added

**Comment by javitonino** [comment]
Working like a charm :rocket:

**Pull request merged**